src/autoTune.ml

```diff
@@ -334,17 +334,16 @@ type option = {
 (*Option for activating the octagon apron domain on selected vars*)
 module VariableMap = Map.Make(CilType.Varinfo)
 module VariableSet = Set.Make(CilType.Varinfo)
+module OctagonTracked = RelationCil.AutotuneTracked
```

Comment:
Also, maybe one needs to rename the option in autotune, because I think it applies globally to all relational domains? (Or at least those built around the shared `CilExp <-> ApronExp` layer?)